Doc/library/shlex.rst

```diff
@@ -36,6 +36,8 @@ The :mod:`shlex` module defines the following functions:
       instance, passing ``None`` for *s* will read the string to split from
       standard input.
 
+   .. deprecated:: 3.8
+      Passing ``None`` for *s*.
```

Comment:
```suggestion
      Passing ``None`` for *s* will raise an exception in future Python versions.
```

This is a bit of nitpicking, which is why I reviewed as "Approved" here, but I think the current "deprecated" text is a bit abrupt. Usually these messages either tell you what to do instead or tell you what the future behavior will be.

Reply:
Okay. Done.